apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -178,6 +185,11 @@ public static IEnumerable<ChannelOption> CreateChannelOptions(this BigtableServi
 
     public partial class BigtableServiceApiClient
     {
+        /// <summary>
+        /// Gets the default settings used with requests.
+        /// </summary>
+        public virtual BigtableServiceApiSettings DefaultSettings => null;
```

Comment:
They hold the defaults for the CallSettings if unspecified for a particular call, so I thought that name was appropriate, but I'm open to changing it if you feel otherwise. I'm not sure what `GetDefaultSettings` is though. Are you referring to `BigtableServiceApiSettings.GetDefault()`? I could see that causing confusion, so I could see the argument for renaming it. Also, it might also make sense to make it internal to avoid confusion altogether. I think I'd just need to adjust how the mocks are setup to return it.